os/booting-on-vmware.md

```diff
@@ -1,71 +1,68 @@
 # Running CoreOS on VMware
 
-CoreOS is currently in heavy development and actively being tested.
-These instructions will walk you through running CoreOS on VMware Fusion or ESXi.
-If you are familiar with another VMware product you can use these instructions as a starting point.
+These instructions will walk you through running CoreOS on VMware Fusion or
+ESXi. If you are familiar with another VMware product you can use these
+instructions as a starting point.
 
 ## Running the VM
 
-These steps will download the VMware image and extract the zip file. After that
-you will need to launch the `coreos_developer_vmware_insecure.vmx` file to create a VM.
-
 ### Choosing a Channel
 
-CoreOS is released into stable, alpha and beta channels. Releases to each channel serve as a release-candidate for the next channel. For example, a bug-free alpha release is promoted bit-for-bit to the beta channel.
-
-The channel is selected based on the URL below. Simply replace `alpha` with `beta`. Read the [release notes]({{site.baseurl}}/releases) for specific features and bug fixes in each channel.
+CoreOS is released into alpha, beta, and stable channels. Releases to each
+channel serve as a release-candidate for the next channel. For example, a
+bug-free alpha release is promoted bit-for-bit to the beta channel.
 
-This is a rough sketch that should work on OS X and Linux:
+The channel is selected based on the URLs below. Simply replace `stable` with
+`alpha` or `beta`. Read the [release notes][release notes] for specific
+features and bug fixes in each channel.
 
 ```sh
-curl -LO http://alpha.release.core-os.net/amd64-usr/current/coreos_production_vmware_insecure.zip
-unzip coreos_production_vmware_insecure.zip -d coreos_production_vmware_insecure
-cd coreos_production_vmware_insecure
-open coreos_production_vmware_insecure.vmx
+curl -LO http://stable.release.core-os.net/amd64-usr/current/coreos_production_vmware_ova.ovf
```

Comment:
no, .ova includes the ovf and vmdk. If you fetch the ovf you do need to fetch the vmdk.